src/PscIde/Command.purs

```diff
@@ -277,9 +277,16 @@ instance decodePursuitCompletion :: DecodeJson PursuitCompletion where
       })
 
 instance decodeImportList :: DecodeJson ImportList where
-  decodeJson json = do
-    imports <- decodeJson json
-    pure (ImportList imports)
+  decodeJson json = decodeObject <|> decodeArray
+    where
+      decodeObject = do
+        o <- decodeJson json
+        moduleName <- o .? "moduleName"
+        imports <- o .? "imports"
+        pure (ImportList { moduleName: head moduleName, imports })
```

Comment:
Why do you need to apply `head` here? Shouldn't the moduleName just be a String?